cmd/gitops/beta/run/cmd.go

```diff
@@ -46,7 +63,14 @@ func RunCommand(opts *config.Options) *cobra.Command {
 gitops beta run . [flags]
 
 # Run the sync against the dev overlay path
-gitops beta run ./deploy/overlays/dev [flags]`,
+gitops beta run ./deploy/overlays/dev
+
+# Run the sync on the dev directory and forward the port. 
+# Listen on port 8080 on all addresses, forwarding to 5000 in a pod of deployment app.
+gitops beta run ./dev --port-forward port=8080:5000,namespace=default,resource=svc/app
```

Comment:
This only listens to 127.0.0.1, not "all addresses". I think it should only listen to localhost (until we add even more flags), so could we change the help text?